Governance Creation (#31)
NASA-AMMOS/seq-json-schema: GOVERNANCE.md

# Multi-Mission Sequence Language Specifications Project Governance

This project aims to create an open source community that encourages transparency, contributions, and collaboration, but maintains sound technical and quality standards. Our goal is to build a community comprised of members across the space community and beyond, including from private organizations, universities, Federally Funded Research and Development Centers (FFRDCs), other government organizations, and international organizations.

This community follows a fairly liberal contribution model where people and/or organizations who do the most work will have the most influence on project direction. Technical decision-making will primarily be made through a "[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)" approach.

## Roles

### User

> Operators who have authored, reviewed, or written tools that use sequence specifications under the scope of this project. These specification were primarily designed for space missions, but let us know if you've found other uses for it.

Reviewer: ```suggestion
Operators who have authored, reviewed, or written tools that use sequence specifications under the scope of this project. These specifications were primarily designed for space missions, but let us know if you've found other uses for them.
```

Author: Fixed!

### Contributor

Contributors include anyone that provides input to the project. This includes code updates, issues, documentation, graphics, designs, or anything else that tangibly improves the project.

### Collaborator

Subset of contributors who have been given write access to one or more of the repositories under the scope of this project. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can review these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner.

> If you are planning on making a substantial contribution or would like to propose a change to our specifications, please follow the [Specification Change Request Process](#specification-change-request-rrocess) below.

Reviewer: ```suggestion
If you are planning on making a substantial contribution or would like to propose a change to our specifications, please follow the [Specification Change Request Process](#specification-change-request-process) below.
```

Author: Fixed!
